src/main/resources/view/MainWindow.fxml

```diff
@@ -23,36 +20,24 @@
         <URL value="@Extensions.css" />
       </stylesheets>
 
-      <VBox>
-        <MenuBar fx:id="menuBar" VBox.vgrow="NEVER">
-          <Menu mnemonicParsing="false" text="File">
-            <MenuItem mnemonicParsing="false" onAction="#handleExit" text="Exit" />
-          </Menu>
```

Comment:
I think there are some trailing white spaces at line 26 and 30, causing the first check to fail.

Reply:
thanks for spotting it bro 👍🏽 resolved! all checks passed now!